app/controllers/application_controller.rb

```diff
@@ -19,6 +20,11 @@ class ApplicationController < ActionController::Base
 
   private
 
+  config.antispam_threshold = 5
```

Comment:
this is set to 5 seconds? hopefully it's short/long enough. @jpastva let us know if you still get spam emails, as we could maybe bump this up. I can't see it taking anyone going straight /contact and speed typing a message in under 30 seconds, but maybe?

Reply:
I'll let you know if they keep coming, thanks!